src/depends/libDatabase/LevelDB.cpp

```diff
@@ -334,8 +344,22 @@ bool LevelDB::BatchInsert(const std::unordered_map<std::string, std::string>& kv
 
     ldb::Status s = m_db->Write(leveldb::WriteOptions(), &batch);
 
-    if (!s.ok())
-    {
+    if (!s.ok()) {
+        return false;
+    }
+
+    return true;
+}
+
+bool LevelDB::BatchDelete(const std::vector<dev::h256>& toDelete) {
+    ldb::WriteBatch batch;
+    for (const auto& i : toDelete) {
+        batch.Delete(leveldb::Slice(i.hex()));
+    }
+
+    ldb::Status s = m_db->Write(leveldb::WriteOptions(), &batch);
+
+    if (!s.ok()) {
```

Comment:
Can use `return s.ok();`
Please make the same improvement in the other functions that return bool too.

Reply:
Added some log under the false case